src/java/org/apache/cassandra/sensors/Type.java

```diff
@@ -29,5 +29,6 @@ public enum Type
     READ_BYTES,
     SEARCH_BYTES,
 
-    WRITE_BYTES;
+    WRITE_BYTES,
+    INDEX_BYTES
```

Comment:
Sure the way you suggested is also more consistent with the other sensors. I was preferring brevity because those would be encoded as custom headers and it is better to keep the mapping 1:1 - however on the second thought, few bytes are negligible and if this turn out to be an issue, we could use a compressed naming format for headers only, no need make the sensor name itself ambiguous...